### PR TITLE
Adding the vundle installation line into the README

### PR DESCRIPTION
**Pull request by jstoja**
For the ones using Vundle :)

### CODE REVIEW

**Comment by michaelghinrichs** [comment]
:+1: plz!

File: README.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ The recommended installation method is via [Pathogen][4].  Then simply execute
     cd ~/.vim/bundle
     git clone https://github.com/mxw/vim-jsx.git
 
+### Vundle
+
+You can add vim-jsx with [Vundle][5]. Just add the following line into your `~/.vimrc`
```
Comment (mxw) [inline README.md]: Two spaces after periods.

File: README.md [hunk 2]
```diff
@@ -20,6 +20,13 @@ The recommended installation method is via [Pathogen][4].  Then simply execute
...
+
+	Bundle 'mxw/vim-jsx'
```
Comment (mxw) [inline README.md]: Use same indent level as other blocks.

**Comment by mxw** [comment]
Okay, okay, you can have your Vundle, once you match the source formatting.

**Comment by jstoja** [comment]
Should be okay now :smile:

**Pull request merged**